### DISCUSSION TITLE
My account is hidden. Why?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#84885] [Projects and Issues]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Hi all, today I faced a problem that my account was hidden from all search engines including github search. I have not performed any manipulations with my account or repositories that are against github rules. I sincerely don't understand what the problem is, please help me figure this out
![image](https://github.com/community/community/assets/116514936/5f2ab08b-31ba-40cb-bf07-e79e8ef9ffe4)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by akpig** [comment]
Hello. I'm facing the same problem as you now. How did you solve it before?
